src/audio/google/google_rtc_audio_processing.c

```diff
@@ -53,9 +53,7 @@ DECLARE_TR_CTX(google_rtc_audio_processing_tr, SOF_UUID(google_rtc_audio_process
 			   LOG_LEVEL_INFO);
 
 struct google_rtc_audio_processing_comp_data {
-#if CONFIG_IPC_MAJOR_4
```

Comment:
agree, I did not touch goole RTC part for ipc3/ipc4 split, because last quarter, a lot of patches are changing this module, if you need me to do this, we can do it later, this module indeed need a separation.